### PR TITLE
Small improvements README.md

### PR DESCRIPTION
**Pull request by aaabramov**

### CODE REVIEW

**Comment by scottf** [comment]
Thank you, but this is not necessary. I'm also happy with the indentation as it is.

File: README.md [hunk 1]
```diff
@@ -769,12 +769,12 @@ in the JetStream examples for a detailed and runnable example.
...
-            Message m = iter.next();
-            // process message
-            m.ack();
-        }
+Iterator<Message> iter = sub.iterate(100, Duration.ofSeconds(1));
+while (iter.hasNext()) {
+    Message m = iter.next();
+    // process message
+    m.ack();
+}
```
Comment (scottf) [inline README.md:777]: The way this is indented is intentional and matches the other indentation in the page. You can see that in the Fetch example just prior to this example. Please remove this change and I will accept the PR.

**Review by scottf — CHANGES_REQUESTED** [review]
Please remove the indent change.

File: README.md (hunk 1 above)
Comment (aaabramov) [reply README.md]: Oh, I see.
Fixed

**Review by scottf — APPROVED** [review]
LGTM

**Pull request merged**